material-stepper/src/main/java/com/stepstone/stepper/StepperLayout.java

```diff
@@ -649,6 +649,61 @@ public void setBackButtonColor(@ColorInt int newButtonColor) {
     }
 
     /**
+     * Changes the color for an active step, applied to dotted or progress bars
+     *
+     * @param newActiveStepColor the active step color integer
+     */
+    public void setActiveStepColor(@ColorInt int newActiveStepColor) {
+        mSelectedColor = newActiveStepColor;
+        mDottedProgressBar.setSelectedColor(newActiveStepColor);
```

Comment:
I think this should be moved to `mStepperType` and handled by each subtype of `AbstractStepperType` instead. This way each stepper type can change it as needed (including the tabbed stepper).

I would also remove 

> applied to dotted or progress bars

from the JavaDoc and implement it for the tabbed stepper instead as the corresponding XML attribute does change the color of the tabs (so that we don't have inconsistencies between XML attribute and Java method).

As described above the logic for that should probably be in `TabsStepperType` and include:
```
        mTabsContainer.setSelectedColor(stepperLayout.getSelectedColor());
```
 \+ what is done in `TabsStepperType#onNewAdapter` (tabs need to be recreated).